phpBB/styles/prosilver/template/viewforum_body.html

```diff
@@ -6,7 +6,13 @@ <h2 class="forum-title"><!-- EVENT viewforum_forum_name_prepend --><a href="{U_V
 <div>
 	<!-- NOTE: remove the style="display: none" when you want to have the forum description on the forum body -->
 	<!-- IF FORUM_DESC --><div style="display: none !important;">{FORUM_DESC}<br /></div><!-- ENDIF -->
-	<!-- IF MODERATORS --><p><strong><!-- IF S_SINGLE_MODERATOR -->{L_MODERATOR}<!-- ELSE -->{L_MODERATORS}<!-- ENDIF -->{L_COLON}</strong> {MODERATORS}</p><!-- ENDIF -->
+	{% if MODERATORS|length %}
+	<p><strong>{% if MODERATORS|length == 1 %}{L_MODERATOR}{% else %}{L_MODERATORS}{% endif %}{L_COLON}</strong>
```

Comment:
Same here for the twig syntax, I would use `{{ lang('') }}` instead of the `L_`, eventhough the rest of the file is phpBB syntax, you are using twig in these lines so might aswell use the language function.

Also, I would put all elements inside the `<p>` on a new line, so the `<strong>` on a new line.